service/history/api/signalwithstartworkflow/signal_with_start_workflow.go

```diff
@@ -151,8 +153,11 @@ func createWorkflowMutationFunction(
 	if currentWorkflowLease == nil {
 		return nil, nil
 	}
-	currentExecutionState := currentWorkflowLease.GetMutableState().GetExecutionState()
+	currentMutableState := currentWorkflowLease.GetMutableState()
+	currentExecutionState := currentMutableState.GetExecutionState()
+	currentWorfklowStartTime := currentMutableState.GetExecutionInfo().StartTime.AsTime()
```

Comment:
```suggestion
	currentWorkflowStartTime := currentMutableState.GetExecutionInfo().StartTime.AsTime()
```

(this typo exists in a few places in this PR, assuming it's not intentional)